repos/system_upgrade/common/models/grubinfo.py

```diff
@@ -0,0 +1,12 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemFactsTopic
+
+
+class GrubInfo(Model):
+    """
+    Information about Grub
+    """
+    topic = SystemFactsTopic
+
+    device_name = fields.Nullable(fields.String())
+    """ Name of the block device where Grub is located. """
```

Comment:
@Rezney  discussed, rather change name to `orig_device_name` and add info into the docstring this is the original name, that is valid only during the preupgrade phases and could be different after the reboot.

```suggestion
    orig_device_name = fields.Nullable(fields.String())
    """
    Original name of the block device where Grub is located.
    
    The name is persistent during the boot of the system so it's safe to be used during
    preupgrade phases. However the name could be different after the reboot, so
    it's recommended to use `leapp.libraries.common.grub.get_grub_device()` anywhere
    else.
    """
```

Reply:
Fixed.